openwisp_users/utils.py

```diff
@@ -57,3 +62,9 @@ def usermodel_list_and_search(model, additional_fields):
         displays = model.list_display[:]
         model.list_display = displays[: field[0]] + [field[1]] + displays[field[0] :]
         model.search_fields += (field[1],)
+
+
+def get_owned_organization(user):
+    return OrganizationOwner.objects.filter(
+        organization_user__in=OrganizationUser.objects.filter(user=user)
+    )
```

Comment:
we can find out if the user is owner of any org efficiently by looping over `user.organization_dicts`, it's explained in [organizations_dict](https://github.com/openwisp/openwisp-users#organizations_dict), the output of that method is cached.

I would add a new helper property (use [cached_property](https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.functional.cached_property)) on the user model for this check, eg: `user.is_owner_of_any_organization`, if this method returns `True` you can then make the change form read only.